apps/ledger-live-mobile/src/screens/Manager/AppsScreen.tsx

```diff
@@ -349,4 +350,10 @@ const AppsScreen = ({
   );
 };
 
+const styles = StyleSheet.create({
+  list: {
+    paddingBottom: 55,
```

Comment:
can you use `TAB_BAR_SAFE_HEIGHT` from `apps/ledger-live-mobile/src/components/TabBar/shared.tsx` instead?

Reply:
Yup, in fact without it it wasn't clickable at all, so good call.